firebase-database/src/androidTest/java/com/google/firebase/database/EventTest.java

```diff
@@ -623,15 +626,15 @@ public void onDataChange(DataSnapshot snapshot) {
               try {
                 IntegrationTestHelpers.waitFor(blockSem);
               } catch (InterruptedException e) {
-                e.printStackTrace();
+                LOGGER.log(WARNING, "unexpected error", e);
```

Comment:
I'll admit I don't fully understand why `printStackTrace()` is considered an anti-pattern. I'm assuming it's because we should be printing to logcat rather than stdout.